modules/compliance-results.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-raw-results.adoc
+
+[id="compliance-results_{context}"]
+= Obtaining Compliance Operator raw results from a Persistent Volume
+
+.Procedure
+
+The Compliance Operator generates and stores the raw results in a Persistent Volume. These results are in Asset Reporting Format (ARF).
+
+. Explore the ComplianceSuite object:
++
+[source,terminal]
+----
+$ oc get compliancesuites nist-moderate -o json \
+| jq '.status.scanStatuses[].resultsStorage'
```

Comment:
should be mentioned `$ suite_name` or compliancesuite name we use in the previous section `i.e nist-moderate-modified`
`$ oc get compliancesuites $suite_name -o json`
OR
`$ oc get compliancesuites nist-moderate-modified -o json`